libs/libcgroup/Makefile

```diff
@@ -0,0 +1,66 @@
+#
+# Copyright (C) 2007-2019 OpenWrt.org
```

Comment:
https://github.com/openwrt/packages/blob/master/CONTRIBUTING.md#makefile-contents-should-contain

Reply:
Is removed